### ISSUE TITLE
Webcam.reset() does not turn off webcam in IE10

### ISSUE THREAD

**Issue by davidrus** [jhuckaby/webcamjs#144]
When I use Webcam.reset() method, my webcam is still active while I close tab or reload page. I need to stop using webcam without page reload or close tab. 

It seems to be bug with all IE browsers... In chrome and another normal browsers is everything ok.

I tried remove element wrapper or set display none, but nothing worked for me. 
Any solutions?

**Comment by davidrus** [comment]
Maybe I found [solution in actionscript](http://stackoverflow.com/questions/1483277/close-webcam-usage-via-actionscript). 

It should be done with call:
```
video.attachCamera(null)
```

**Comment by davidrus** [comment]
I created PR for this https://github.com/jhuckaby/webcamjs/pull/145 which fixed this issue for me.

**Comment by davidrus** [comment]
Thanks for merge my PR, can You release new version and after that close this issue?

**Comment by jhuckaby** [comment]
Yes, but I have to test it everywhere first.  I'm a little bit worried because this is a bug fix only for IE 11, but the releaseCamera code runs everywhere on all browsers and platforms that use Flash.

**Comment by davidrus** [comment]
Yeah, thank You for info :). But I think it's good to release camera in every devices which use flash. It's just like "destructor" for camera.

**Comment by gggg990** [comment]
@jhuckaby Did you perhaps find a solution for this?
